controllers/machinehealthcheck_controller.go

```diff
@@ -75,7 +75,7 @@ type MachineHealthCheckReconciler struct {
 }
 
 func (r *MachineHealthCheckReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, options controller.Options) error {
-	controller, err := ctrl.NewControllerManagedBy(mgr).
+	managedController, err := ctrl.NewControllerManagedBy(mgr).
```

Comment:
Please revert this. 

Looking at the proposed changes, even though the variable collides with the package name, it reads much better as it is today. Moreover, it's consistent with the rest of the codebase and the controller package isn't used within this method.